src/backend/web/validation.js

```diff
@@ -0,0 +1,26 @@
+const Ajv = require('ajv');
+
+const feedSchema = {
+  properties: {
+    user: { type: 'string', minLength: 1 },
+    url: { type: 'string', pattern: '^http[s]?:/{2}(www)?\\.?.+\\..+$', minLength: 1 },
```

Comment:
@humphd : ah, good point. I completely did not see it.

Reply:
@humphd : if I used `"format": "uri"` , it would still require pattern to match just http(s). However, there is `"format": "url"` which is referred as deprecated but we can still use it and it only include http(s) scheme. Which direction would you prefer?

Reply:
I don't want to use a pattern for URL detection here, since we'll get it wrong and cause problems, so if it's a valid URI, that's probably enough.

Reply:
Then I think using ` "format" : "url"` would be better because ` "format": "uri"` will match sth like chrome://www.google.com which is not what we want. In the new commit, i actually use  ` "format" : "url"`

Reply:
If you want to use 'uri' then we need to include ` "pattern" : "^http[s]?"` so we can just match http(s) scheme not sth like chrome

Reply:
Another way to do this is to define our own Ajv custom keyword, for example: https://gist.github.com/moshfeu/a268788ba1c57b8c8da453c7d05008ea#file-final-code-js.

This example is trying to limit `uri` to `https`.  We could do something similar but limit to `http` or `https`.

Reply:
@humphd : great! Im on it!